custom_code/templatetags/target_list_extras.py

```diff
@@ -13,3 +17,20 @@ def target_list_extra_field(target_list, name):
         return TargetListExtra.objects.get(target_list=target_list, key=name).value
     except TargetListExtra.DoesNotExist:
         return None
+
+@register.inclusion_tag('tom_targets/partials/galaxy_table.html')
+def galaxy_table(target):
+    """
+    Displays the most likely host galaxy matches.
+    """
+    extras = {k['name']: target.extra_fields.get(k['name'], '') for k in settings.EXTRA_FIELDS if not k.get('hidden')}
+    return {
+        'target': target,
+        'extras': extras
+    }
```

Comment:
I can't tell why this is needed, because I think you never use the `extras` variable on the galaxy tab. But maybe I'm missing something.

If it is needed, we should give a better description in the docstring. This really returns all the target extras, not just the galaxy-related ones.

Reply:
Sorry, this comment is completely wrong. Ignore me! It's true that the `extras` are not used though.